test/integration/index.html

```diff
@@ -11,6 +11,7 @@
     <script>
         // These variables are going to be available in the "window global" intentionally.
         // Allows you easy access to debug with `vm.greenFlag()` etc.
+        window.devicePixelRatio = 1;
```

Comment:
Why?

Reply:
just to force a specific value, we could lock it to 2 if we wanted instead?

Reply:
1 is probably more common right now, so it probably makes sense for our tests. Does setting this actually impact the way the canvas is rendered?